### PR TITLE
Added: Contact Page & Learn Cloud Native: Managing service

### LINKED ISSUE #711: [NG] Migrate the site data for the Contact page
**Description**
Currently, the page dedicated to [Contact](https://layer5ng.netlify.app/contact) on the [layer5-ng site](https://layer5ng.netlify.app/) points to a 404 error. 

**Expected Behavior**

The data from the [Contact](https://layer5.io/contact) on the [layer5.io site](https://layer5.io/) needs to be migrated to the layer5-ng site. 

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by kishore007k**
Signed-off-by: kishore007k <kishorekumarstar07@gmail.com>

**Description**

The Contact page from the [layer5/contact](https://layer5.io/contact) is migrated to the [layer5-ng/contact](https://layer5ng.netlify.app/contact)
And also the `Learn Cloud Native: Managing service meshes with Meshery news article` is added to the [layer5-ng/news](https://layer5ng.netlify.app/news).

This PR fixes #711 and #832 

**Notes for Reviewers**
@Nikhil-Ladha 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Contact/contact-content.js [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+// Icons
+import { BsThreeDots } from "react-icons/bs";
+
+// Images
+import Jobs_White from "../../images/contact/jobs_white.svg";
+import Support_White from "../../images/contact/support_white.svg";
+import Partner_White from "../../images/contact/partner_white.svg";
+import Contact_White from "../../images/contact/contact_white.svg";
+
+const Wrapper = styled.div`
```
Comment (Jashpatel1) [inline src/sections/Contact/contact-content.js:14]: Can you please create a separate style file and import the wrapper from there?

File: src/sections/Contact/contact-content.js [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+      <div className="container">
+
+        <div className="card">
+          <div className="cardsContainer">
+            <Link to="/career" className="image-container">
+            <img src={Jobs_White} alt="jobs_white" className="image" />
+            </Link>
+            <div className="text-and-button">
+              <h3 className="title">Careers</h3>
+              <Link to="/career">
```
Comment (Jashpatel1) [inline src/sections/Contact/contact-content.js:177]: The link should be /careers

File: src/sections/Contact/contact-content.js (hunk 1 above)
Comment (kishore007k) [reply src/sections/Contact/contact-content.js:14]: Sure!

File: src/sections/Contact/contact-content.js [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+`;
+
+const ContactContent = () => {
+  return (
+    <Wrapper>
+      <div className="container">
+
+        <div className="card">
+          <div className="cardsContainer">
+            <Link to="/career" className="image-container">
```
Comment (Jashpatel1) [inline src/sections/Contact/contact-content.js:172]: The link should be /careers

File: src/sections/Contact/contact-content.js [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+                <BsThreeDots className="three-dots" />
+              </Link>
+            </div>
+            <p className="para">I'm interested in career opportunities in Layer5</p>
+          </div>
+        </div>
+
+        <div className="card">
+          <div className="cardsContainer">
+            <Link to="/support" className="image-container">
```
Comment (Jashpatel1) [inline src/sections/Contact/contact-content.js:187]: There should be an href to mailto:support@layer5.io

File: src/sections/Contact/contact-content.js [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+              <img src={Support_White} alt="support_white" className="image" />
+            </Link>
+            <div className="text-and-button">
+              <h3 className="title">Support</h3>
+              <Link to="/career">
```
Comment (Jashpatel1) [inline src/sections/Contact/contact-content.js:192]: There should be an href to mailto:support@layer5.io

File: src/sections/Contact/contact-content.js [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+                <BsThreeDots className="three-dots" />
+              </Link>
+            </div>
+            <p className="para">I need assistance with my cloud native infrastructure.</p>
+          </div>
+        </div>
+
+        <div className="card">
+          <div className="cardsContainer">
+            <Link to="/partnership" className="image-container">
```
Comment (Jashpatel1) [inline src/sections/Contact/contact-content.js:202]: The link should be /partners

File: src/sections/Contact/contact-content.js [hunk 7]
```diff
@@ -0,0 +1,235 @@
...
+              <img src={Partner_White} alt="partner_white" className="image" />
+            </Link>
+            <div className="text-and-button">
+              <h3 className="title">Partnership</h3>
+              <Link to="/career">
```
Comment (Jashpatel1) [inline src/sections/Contact/contact-content.js:207]: The link should be /partners

File: src/sections/Contact/contact-content.js [hunk 8]
```diff
@@ -0,0 +1,235 @@
...
+        </div>
+
+        <div className="card">
+          <div className="cardsContainer">
+            <Link to="/about" className="image-container">
+              <img src={Contact_White} alt="contact_white" className="image" />
+            </Link>
+            <div className="text-and-button">
+              <h3 className="title">Contact</h3>
+              <Link to="/career">
```
Comment (Jashpatel1) [inline src/sections/Contact/contact-content.js:222]: The link should be /about

File: src/sections/Contact/contact-content.js [hunk 9]
```diff
@@ -0,0 +1,235 @@
...
+    .three-dots{
+      width: 30px;
+    }
+  }
+`;
+
+const ContactContent = () => {
+  return (
+    <Wrapper>
+      <div className="container">
```
Comment (Tanuj22) [inline src/sections/Contact/contact-content.js:168]: Can you use `Container`, `Row`, and `Col` components already defined? This is basically recreating the work that has already been done

File: src/sections/Contact/contact-content.js [hunk 10]
```diff
@@ -0,0 +1,235 @@
...
+
+        <div className="card">
+          <div className="cardsContainer">
```
Comment (Tanuj22) [inline src/sections/Contact/contact-content.js:171]: Can you create a separate component for this card called `ContactCard` and keep it in the `components` folder and use that here (passing the title, image, and link as props to that component).

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Good work @kishore007k! Just some code clean-up needed.

File: src/sections/Contact/contact-content.js (hunk 4 above)
Comment (kishore007k) [reply src/sections/Contact/contact-content.js:187]: instead of `/support` in the `href` I should replace it with the `mailto:support@layer5.io` ?

Comment (Jashpatel1) [reply src/sections/Contact/contact-content.js:187]: Yes, but you will have to use <a href="mailto:support@layer5.io"> tag I guess.

Comment (kishore007k) [reply src/sections/Contact/contact-content.js:187]: But then I can't use the `Link` tag. So, should I have to change it to `a` tag instead?

Comment (Jashpatel1) [reply src/sections/Contact/contact-content.js:187]: > But then I can't use the `Link` tag. So, should I have to change it to `a` tag instead?

Yes

File: src/components/ContactCard/ContactCard.js [hunk 11]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Link } from "gatsby";
```
Comment (Nikhil-Ladha) [inline src/components/ContactCard/ContactCard.js:2]: Better to fix this warning.

**Comment by leecalcote** [comment]
Thank you for working through this, @kishore007k. It'd be good to ensure that we transfer to the datacenter image over as well (the guy pushing over server racks) - 

Screen Shot 2020-10-04 at 7 23 54 PM

File: src/sections/Contact/contact-content.js [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+import { Container, Col, Row} from "../../reusecore/Layout";
+import ContactCardStyle from "./contactcard.style";
+
+
+const ContactContent = () => {
+  return (
+  <ContactCardStyle>
+    <Container>
+      <Row Hcenter>
+        <Col xs={12} sm={12} md={6} lg={6} xl={6}>
```
Comment (Jashpatel1) [inline src/sections/Contact/contact-content.js:19]: The 3 cards in a row was a better configuration for md devices and as Lee stated you will require space in the second row to incorporate the graphic. So please change this to 3 cards in a row.

File: src/components/ContactCard/ContactCard.js (hunk 11 above)
Comment (kishore007k) [reply src/components/ContactCard/ContactCard.js:2]: Can you elaborate on what needs to be done? I don't quite understand.

File: src/sections/Contact/contact-content.js (hunk 12 above)
Comment (kishore007k) [reply src/sections/Contact/contact-content.js:19]: Ok sure will do that right away.

File: src/components/ContactCard/ContactCard.js (hunk 11 above)
Comment (Tanuj22) [reply src/components/ContactCard/ContactCard.js:2]: Just remove this unused import. Warning should be fixed then

**Review by Jashpatel1 — APPROVED** [review]
Looks good to me. Nice work here, Thanks @kishore007k

**Comment by kishore007k** [comment]
> Looks good to me. Nice work here, Thanks @kishore007k

Thanks @Jashpatel1

**Review by Nikhil-Ladha — APPROVED** [review]
Looks good to me 👌🏼

**Review by Tanuj22 — APPROVED** [review]
Nice Work!

**Pull request merged**